selfdrive/car/hyundai/interface.py

```diff
@@ -69,7 +69,11 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
 
       # These cars use the FCA11 message for the AEB and FCW signals, all others use SCC12
       if 0x38d in fingerprint[0] or 0x38d in fingerprint[2]:
-        ret.flags |= HyundaiFlags.USE_FCA.value
+        ret.flags |= HyundaiFlags.USE_FCA11.value
+
+      # These cars use the FCA12 message for the status of AEB and FCW, all others use SCC13a
+      if 0x483 in fingerprint[0] or 0x483 in fingerprint[2]:
+        ret.flags |= HyundaiFlags.USE_FCA12.value
```

Comment:
Any time we add a new feature detection, we need to do some minimum validation to ensure the message we're checking starts up quick enough to always be detected (ideally 2x faster than our CAN fingerprint gathering time)

Reply:
For reference, we had some issues detecting the gas pedal signal on Toyota as sometimes it took 1 second to start sending, but we gather CAN fingerprints for less than that (and we want to reduce it soon)